task/manager.go

```diff
@@ -992,33 +1003,12 @@ func (m *Manager) deleteOrphanPods() {
 // Includes:
 //   - pod YAML
 //   - pod Events
-//   - container Logs
 func (m *Manager) podSnapshot(task *Task, pod *core.Pod) (err error) {
-	var files []*model.File
-	d, err := m.podYAML(pod)
-	if err != nil {
-		return
-	}
-	files = append(files, d)
-	logs, err := m.podLogs(pod)
-	if err != nil {
-		return
-	}
-	files = append(files, logs...)
-	for _, f := range files {
-		task.attach(f)
-	}
-	Log.V(1).Info("Task pod snapshot attached.", "id", task.ID)
-	return
-}
-
-// podYAML builds pod resource description.
-func (m *Manager) podYAML(pod *core.Pod) (file *model.File, err error) {
```

Comment:
**hint**: The podYAML() can be rolled into podSnapshot() since we're no longer collecting logs here.